demo/.eslintrc.js

```diff
@@ -14,8 +14,9 @@ module.exports = {
     "prettier/react",
   ],
   rules: {
-    "no-console": 0,
-    "react/prop-types": 0,
+    "no-console": "off",
+    "react-native/no-raw-text": "off",
+    "react/prop-types": "off",
```

Comment:
I think we want `react/prop-types` enabled in the long run. But either way, this change to strings for the config rather than integers is a great change!

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md